src/renderer/getDiForUnitTesting.tsx

```diff
@@ -103,8 +111,35 @@ export const getDiForUnitTesting = (opts: { doGeneralOverrides?: boolean } = {})
 
     di.override(lensResourcesDirInjectable, () => "/irrelevant");
 
-    di.override(watchHistoryStateInjectable, () => () => () => {});
+    // TODO: Remove side-effects and shared global state
+    di.override(commandContainerRootFrameChildComponentInjectable, () => ({
```

Comment:
what about the other root frame child components? They don't need overriding?

Reply:
No, `commandContainer` seem to be only one.